content/guides/component-testing/vue/examples.md

```diff
@@ -58,8 +58,11 @@ prop and validate it was called:
 it('clicking + fires a change event with the incremented value', () => {
   const onChangeSpy = cy.spy().as('onChangeSpy')
   cy.mount(Stepper, { props: { onChange: onChangeSpy } })
-  cy.get('[data-cy=increment]').click()
-  cy.get('@onChangeSpy').should('have.been.calledWith', 1)
+  cy.get('[data-cy=increment]')
+    .click()
+    .then(() => {
```

Comment:
Could you revert this as well, and the other two parts of the diff that add `.then()`? Let's keep just the text update to call out that `eventName` emitted from the component becomes `onEventName` in the props when using the component.

Reply:
Yes, of course. My bad!